client/src/app/pages/issues/issue-detail-drawer/file-incidents-detail-modal/incident-code-snip-viewer.tsx

```diff
@@ -42,6 +52,25 @@ export const IncidentCodeSnipViewer: React.FC<IIncidentCodeSnipViewerProps> = ({
       ] as Language)
     : undefined;
 
+  if (
+    incident.codeSnip === undefined ||
+    incident.codeSnip === null ||
+    incident.codeSnip == ""
```

Comment:
Any chance the `codeSnip` could be a blank strings like `"         "`?

Maybe throw in a `incident.codeSnip.trim()` just for completeness?

This makes me miss the old Java apache commons string `isNotBlankString(s)` utility method.